src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatSymb = fromFormat.pop();
```

Comment:
Avoid modifying the input parameters, create a new variable to store the last element of 'fromFormat' and 'toFormat' arrays.